blaze/expr/tests/test_arithmetic.py

```diff
@@ -72,3 +72,15 @@ def test_dir():
 def test_arith_ops_promote_dtype():
     r = w + z
     assert r.dshape == dshape('5 * 3 * float64')
+
+
+def test_str_arith():
+    assert isinstance(cs + cs, Concat)
+    assert isinstance(cs * 1, Repeat)
+    assert isinstance(cs % cs, Interp)
+
+    with pytest.raises(Exception):
```

Comment:
can you add a test against scalars for add and interp?

something that tests expressions like this:

```python
cs + 'a'
cs % 'a'
```